spec/factories/users.rb

```diff
@@ -6,5 +6,9 @@
     first_name { 'John' }
     last_name { 'Doe' }
     password { 'password-1234' }
+
+    trait :confirmed do
```

Comment:
We could default users to being confirmed, but IMO assuming they are unconfirmed by default is safer.